lib/simplifyExpression/multiplyFractionsSearch/index.js

```diff
@@ -23,13 +25,17 @@ function multiplyFractions(node) {
     return Node.Status.noChange(node);
   }
   const atLeastOneFraction = node.args.some(
-    arg => Node.Type.isOperator(arg, '/'));
-  const hasPolynomialTerms = node.args.some(
-    arg => Node.PolynomialTerm.isPolynomialTerm(arg));
-  if (!atLeastOneFraction || hasPolynomialTerms) {
+      arg => Node.Type.isOperator(arg, '/'));
+
+  // If term is in the form of <constant fraction> <polynomial term>
+  // do not multiply in order to keep pedagogical style
+  // e.g 1/2 * x^2 -> 1/2 x^2 and not 1x^2 / 2
```

Comment:
also what if you had `x^2 * 1/2` ? I feel like you wouldn't want to multiply it out there either